.github/CODEOWNERS

```diff
@@ -1,316 +1,16 @@
-/tools/public_api_guard/material/icon**                     @andrewseguin
-/tools/public_api_guard/material/input**                    @devversion @mmalerba
-/tools/public_api_guard/material/list**                     @mmalerba @devversion
-/tools/public_api_guard/material/menu**                     @crisbeto
-/tools/public_api_guard/material/paginator**                @crisbeto
-/tools/public_api_guard/material/prebuilt-themes**          @andrewseguin
-/tools/public_api_guard/material/progress-bar**             @andrewseguin
-/tools/public_api_guard/material/progress-spinner**         @andrewseguin
-/tools/public_api_guard/material/radio**                    @mmalerba
-/tools/public_api_guard/material/schematics**               @devversion @andrewseguin
-/tools/public_api_guard/material/select**                   @crisbeto
-/tools/public_api_guard/material/select**                   @crisbeto
-/tools/public_api_guard/material/sidenav**                  @mmalerba
-/tools/public_api_guard/material/slide-toggle**             @crisbeto
-/tools/public_api_guard/material/slider**                   @devversion
-/tools/public_api_guard/material/snack-bar**                @andrewseguin
-/tools/public_api_guard/material/sort**                     @andrewseguin
-/tools/public_api_guard/material/stepper**                  @mmalerba
-/tools/public_api_guard/material/table**                    @andrewseguin
-/tools/public_api_guard/material/tabs**                     @crisbeto
-/tools/public_api_guard/material/testing**                  @andrewseguin
-/tools/public_api_guard/material/toolbar**                  @devversion
-/tools/public_api_guard/material/tooltip**                  @andrewseguin
-/tools/public_api_guard/material/tree**                     @jelbourn @andrewseguin
-
-/tools/public_api_guard/youtube-player/**                   @crisbeto
-/tools/public_api_guard/google-maps/**                      @crisbeto
-
-# Misc
-/.github/**                                                 @devversion @josephperrott
-/.husky/**                                                  @devversion @josephperrott
-/.github/CODEOWNERS                                         @devversion @josephperrott @andrewseguin @jelbourn
-/.github/ISSUE_TEMPLATE/**                                  @andrewseguin @jelbourn
-/.vscode/**                                                 @devversion @josephperrott @mmalerba
-/.ng-dev/**                                                 @devversion @josephperrott
-/goldens/size-test.yml                                      @andrewseguin @mmalerba @crisbeto
-/goldens/**                                                 @andrewseguin @mmalerba @crisbeto
-/src/*                                                      @devversion @josephperrott
-/*                                                          @devversion @josephperrott
+*                                   @angular/components-googlers
```

Comment:
I think you actually need to have the base directories you want to include here.

`*` would just refer to the files at the root of the repository

`**` would be everything in the repo, but that would include the other files I believe.

Reply:
I think `*` might be a special case, cause I do see that something like `dir/*` only covers immediate folder files (not nested). Here's what their [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) say

```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2
```

Reply:
oh okay, works for me!